src/site/content/en/blog/layoutNG/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: LayoutNG
+subhead: |
+  A new layout engine for Chromium
+authors:
+  - eae
+date: 2019-05-29
+hero: hero.jpg
+alt: hand sketched diagram of Chrome's paint and render pipeline
+description: |
+  LayoutNG is a new layout engine for Chromium that has been designed for the needs of modern scalable web applications. It provides improved performance isolation, better support for scripts other than Latin, fixes many issues around floats/margins and a large number of web compatibility issues.
+tags:
+  - post
+  - layout
+---
+
+<style>
+  .w-figures-2-up {
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-start;
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: repeat(auto-fit, minmax(20ch, 1fr));
+  }
+</style>
+
+Scheduled to be released in Chrome 76, LayoutNG is a new layout engine exiting a multi-year effort. While this article intends to cover the list of immediate effects, longer term goals exist which intend to bring additional performance gains and advanced layout features.
+
+<h2 class="w-headline--four">What's New?</h2>
```

Comment:
I've proposed promoting those to h3 by removing the "Developer visible changes" h2, as noted below. But either way, I guess I'd be in favor of retaining the default styling of heads because they convey the IA of the post. (In this case, "What's new?" is a separate top-level section from what comes after.) But I'll defer to you :)